README.md

```diff
@@ -20,104 +38,162 @@ for the following use cases:
 
-[quickstart]: https://aws.amazon.com/quickstart/architecture/heptio-kubernetes/
+* An admin `kubeconfig` file, and the KUBECONFIG environment variable set.
 
-* `kubectl` installed. See [installing via Homebrew (MacOS)][brew] or [building
+* For some advanced workflows it may be required to have `kubectl` installed. See [installing via Homebrew (MacOS)][brew] or [building
   the binary (Linux)][linux].
 
-* An admin `kubeconfig` file, and the KUBECONFIG environment variable set.
+## Installing
 
-[brew]: https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-with-homebrew-on-macos
-[linux]: https://kubernetes.io/docs/tasks/tools/install-kubectl/#tabset-1
+Download one of the releases directly from [here][releases].
 
-## Using the CLI
+Alternatively, you can install the CLI by running:
 
-Sonobuoy also provides a CLI that lets you run Sonobuoy on your cluster. By default, the CLI
-records the following results:
+```
+go get -u -v github.com/heptio/sonobuoy
+```
 
-* Information about your cluster's hosts, Kubernetes resources, and versions.
-* systemd logs from each host. Requires a plugin.
-* The results of a e2e conformance tests.
+Golang version 1.12 or greater is recommended. Golang can be installed via
+[gimme][gimme].
 
-### CLI Prerequisites
+## Getting Started
+To launch Sonobuoy for a short run and wait until it is finished run:
+```
+sonobuoy run --mode quick --wait
+```
 
-* Golang installed. We recommend [gimme][gimme], with golang version 1.12.0.
+Get the results from the plugins (e.g. e2e test results):
+```
+results=$(sonobuoy retrieve)
+```
 
-* Your $PATH configured:
+Remove the Sonobuoy pod and wait for the namespace to be removed:
+```
+sonobuoy delete --wait
+```
 
+Inspect results for test failures.  This will list the number of tests failed and their names:
+```
+sonobuoy e2e $results
 ```
-$ export PATH=$GOROOT/bin:$GOPATH/bin:$PATH 
-```  
 
-[gimme]: https://github.com/travis-ci/gimme
+### Checking Cluster Conformance
+
+To run Sonobuoy to check if your cluster passes Kubernetes conformance, run:
+```
+sonobuoy run
+```
+
+This is the command which will generate the data necessary to validate your
+cluster is conformant according to the [CNCF][cncf].
 
-### Download and run
+### Quick Mode
 
-Download the CLI by running:
+Often you don't want to wait for an entire conformance run to complete but want
+some minimal check that the cluster is behaving and you have the proper
+configurations in place to run conformance in the future.
 
+In that case, run:
 ```
-$ go get -u -v github.com/heptio/sonobuoy
+sonobuoy run --mode quick
 ```
 
-Deploy a Sonobuoy pod to your cluster with:
+This runs only the e2e plugin (Kubernetes tests) but specifies that it should
+only run a single, short test. This should normally take less than a minute to complete.
 
+### Monitoring Sonobuoy during a run
+You can check on the status of each of the plugins running with:
 ```
-$ sonobuoy run
+sonobuoy status
```

Comment:
See https://github.com/civicrm/civicrm-dev-docs/issues/190 for a random discussion of `$` prefix vs not. I think they hit on the relevant points and I liked where they landed.

tl;dr; No prefix unless mixing input/output where you need to distinguish them.

Reply:
I always put in the prefix and on your issue, they revised to always use: https://github.com/civicrm/civicrm-dev-docs/issues/190#issuecomment-322019933 

I don't feel super strongly either way, but like how putting it in calls out that it's something you run in a terminal.

Reply:
Haha, apparently I stopped reading at the first summary when it matched my feelings 😅 I can do either; I wish github markdown had a way to put a non selectable image there instead of text so you could see it but it wouldn’t bother copy paste.

Reply:
haha, we just need to be consistent. Looks like velero doesn't use them, but I like how it looks when you show a command with output, to distinguish the command from the output. ¯\_(ツ)_/¯

Reply:
👍 I'm at least being consistent here; not using them throughout. I only do show single line commands and am not mixing with output so it doesnt really matter in the end now that I've fixed up some other things.